openupgrade_scripts/scripts/mass_mailing/15.0.2.5/pre-migration.py

```diff
@@ -0,0 +1,147 @@
+from openupgradelib import openupgrade
+
+
+def _rename_fields(env):
+    openupgrade.rename_columns(
+        env.cr,
+        {
+            "mailing_mailing": [
+                ("contact_ab_pc", "ab_testing_pc"),
+                ("unique_ab_testing", "ab_testing_enabled"),
+            ],
+            "mailing_trace": [
+                ("sent", "sent_datetime"),
+                ("opened", "open_datetime"),
+                ("replied", "reply_datetime"),
+                ("clicked", "links_click_datetime"),
+            ],
+        },
+    )
+
+
+def _change_value_selection(env):
```

Comment:
@daonamutc1 tên hàm nên đặt theo định dạng `_<chức năng>_<model>_<field>`
```suggestion
def _map_mailing_mailing_reply_to_mode(env):
```